src/org/zaproxy/zap/view/MainToolbarPanel.java

```diff
@@ -178,6 +178,9 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 						default: return;	// Not recognised
 					}
 					Control.getSingleton().setMode(mode);
+					if (View.isInitialised()){
```

Comment:
The if statement can be removed, the view is initialised (it's view code that will call this method).